test/over_react/component/shared_error_boundary_tests.dart

```diff
@@ -9,8 +9,10 @@ import './fixtures/flawed_component_on_mount.dart';
 import './fixtures/flawed_component_that_renders_a_string.dart';
 import './fixtures/flawed_component_that_renders_nothing.dart';
 
-void sharedErrorBoundaryTests(BuilderOnlyUiFactory builder) {
-  TestJacket<ErrorBoundaryMixin> jacket;
+/// `isWrapper` identifies an ErrorBoundary that wraps another Error Boundary in order to handle
```

Comment:
#nit this should be a doc comment reference
```suggestion
/// [isWrapper] identifies an ErrorBoundary that wraps another Error Boundary in order to handle
```